.github/workflows/packages.yml

```
@@ -66,29 +66,30 @@ jobs:
           with:
             version: nightly
 
-        - name: Verify Changed files
-          uses: tj-actions/verify-changed-files@v11.1
-          id: verify-yarn-lock
+        - name: Filter changed files
+          uses: dorny/paths-filter@v2
```

actionlint (1.7.4)

the runner of "dorny/paths-filter@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)